packages/react/accordion/src/Accordion.tsx

```diff
@@ -254,37 +270,56 @@ const AccordionImpl = React.forwardRef<AccordionImplElement, AccordionImplProps>
       event.preventDefault();
 
       let nextIndex = triggerIndex;
+      const homeIndex = 0;
+      const endIndex = triggerCount - 1;
+
+      const moveNext = () => {
+        nextIndex = triggerIndex + 1;
+        if (nextIndex > endIndex) {
+          nextIndex = homeIndex;
+        }
+      };
+
+      const movePrev = () => {
+        nextIndex = triggerIndex - 1;
+        if (nextIndex < homeIndex) {
+          nextIndex = endIndex;
+        }
+      };
```

Comment:
It might be nice for consistency to get closer to how we generally handle direction aware arrow keys. Maybe closer to what we do in `Slider` potentially?

Reply:
I'm not following this 100%. Are you suggesting we add an OrientationProvider and pass down separate implementations for each key based on the orientation?

The functions you highlighted here are just helpers to avoid having duplicate code and are agnostic of direction. The direction logic only applies to `orientation="horizontal"` and `isDirectionLTR`

Reply:
On second thoughts, I think this is fine for now as it does the job and we will potentially rework this using `focusFirst` approach as suggested by @andy-hook here: https://github.com/radix-ui/primitives/pull/1850#discussion_r1053136773